README.md

```diff
@@ -32,6 +31,10 @@ The Scheduler is the main part of the language. It parses PFDL files and generat
 
 For more infos visit the official :books: [Documentation](https://iml130.github.io/pfdl/).
 
+## Quick Installation
+The fastest way to get the PFDL Scheduler running is by installing the [PyPi package](https://pypi.org/project/pfdl-scheduler/) via pip:
+> pip install pfdl-scheduler
```

Comment:
If you already add `.venv` to the `.gitignore` you probably want to point users to installing the package into a venv instead of suggesting to mess up the entire system wide installation ;)

Reply:
Dont you think that this is up to the user? :D

Reply:
I don't think it hurts to just include it in the default install instructions, for example to change them to

```suggestion
The fastest way to get the PFDL Scheduler running is by installing the [PyPi package](https://pypi.org/project/pfdl-scheduler/) via pip in a virtual environment:
> python3 -m venv venv-pfdl
> source venv-pfdl/bin/activate
> pip install pfdl-scheduler
```

Reply:
Done, i also added the link to the venv documentation.